src/index.html

```diff
@@ -7,6 +7,358 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+          class="header__logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#apple" class="header__nav-link is-active">apple</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__nav-link">samsung</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#smartphones" class="header__nav-link">smartphones</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+             href="#laptops&computers"
+             class="header__nav-link"
+             data-qa="nav-hover"
+            >
+                laptops & computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#gadgets" class="header__nav-link">gadgets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#tablets" class="header__nav-link">tablets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#photo" class="header__nav-link">photo</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="#video" class="header__nav-link">video</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main class="product-grid">
+      <div class="product-grid__container">
+        <article class="product-card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27inch Retina 5K"
+            class="product-card__picture"
+          >
+          <h2 class="product-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__code">Product code: 195434</p>
+
+          <div class="product-card__reviews">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="product-card__reviews-text">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-label">Price:</p>
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+
+          <a
+          href="#buy"
+          class="product-card__link"
+          data-qa="card-hover">
+          buy
+          </a>
```

Comment:
also here, check in all places

Reply:
<a
            href="#buy"
            class="product-card__link"
            data-qa="card-hover">
            buy
          </a>

Made it look like that in all instances